packages/core/src/routes/phrase.ts

```diff
@@ -0,0 +1,98 @@
+import resource from '@logto/phrases-ui';
+import { CustomPhrases } from '@logto/schemas';
+import { adminConsoleApplicationId, adminConsoleSignInExperience } from '@logto/schemas/lib/seeds';
+import { LanguageKey, languageKeyGuard } from '@logto/shared';
+import deepmerge from 'deepmerge';
+import { ResourceLanguage } from 'i18next';
+import { errors, Provider } from 'oidc-provider';
+
+import RequestError from '@/errors/RequestError';
+import detectLanguage from '@/i18n/detect-language';
+import koaGuard from '@/middleware/koa-guard';
+import { findAllCustomLanguageKeys, findCustomPhraseByLanguageKey } from '@/queries/custom-phrase';
+import { findDefaultSignInExperience } from '@/queries/sign-in-experience';
+
+import { AnonymousRouter } from './types';
+
+const getLanguageInfo = async (applicationId: unknown) => {
```

Comment:
```suggestion
const getLanguageInfo = async (applicationId: string) => {
```
Why is this type `unknown`?

Reply:
Since the type of applicationId from the Interaction is `unknown`, use it to pass the type check.

<img width="890" alt="image" src="https://user-images.githubusercontent.com/10594507/190604743-2e4ff61a-77c6-49c5-882e-931260c7e43a.png">

Reply:
see #1959